aws/src/database/dynamodb/scripts/asset_table/get_assets_add_patchId_areaId.py

```diff
@@ -0,0 +1,138 @@
+from dataclasses import asdict, dataclass
+
+from mypy_boto3_dynamodb.service_resource import Table
+from boto3.dynamodb.types import TypeDeserializer
+import json
+
+from aws.src.database.domain.dynamo_domain_objects import Asset
+from aws.src.database.dynamodb.utils.get_dynamodb_table import get_dynamodb_table
+from aws.src.utils.progress_bar import ProgressBar
+from aws.src.utils.logger import Logger
+from enums.enums import Stage
+from pathlib import Path
+
+from utils.confirm import confirm
+
+
```

Comment:
Would should add a comment somewhere to let devs know that we exported the asset table to S3 and used that for this script.

Reply:
Cheers, added to the PR details

Reply:
I think the PR details are a good place to add it but we should also add it to the script as well so that if devs ever need to run this script (or something similar)  they can easily know how we did it.

Reply:
Added a module docstring in 44557a21e753691134980e835986fe347b76d4e2